rsa/prime.py

```diff
@@ -103,13 +90,32 @@ def randomized_primality_testing(n, k):
 def is_prime(number):
     """Returns True if the number is prime, and False otherwise.
 
+    >>> is_prime(2)
+    True
     >>> is_prime(42)
     False
     >>> is_prime(41)
     True
+    >>> [x for x in range(901, 1000) if is_prime(x)]
+    [907, 911, 919, 929, 937, 941, 947, 953, 967, 971, 977, 983, 991, 997]
     """
 
-    return randomized_primality_testing(number, 6)
+    # Check for small numbers.
+    if number < 10:
+        return number in [2, 3, 5, 7]
```

Comment:
Never build a list for this. Using a set literal is 33% faster (on my simple test) -- the percentage can change, but a set test is faster than a list test. I'll handle this in another commit.